File: testers/grpc/scala/generated-and-checked-in/com/example/grpc/CustomerId.scala
package com.example.grpc



/** Wrapper type for typr.grpc.GrpcCodegen$$$Lambda/0x00007fc001118400@6c4906d3 */
case class CustomerId(value: String) extends scala.AnyVal {
  /** Get the underlying value */
  def unwrap: String = {
    return this.value
  }
}

object CustomerId {
  /** Create a CustomerId from a raw value */
  def valueOf(v: String): CustomerId = {
    return new CustomerId(v)
  }
}

File: testers/grpc/kotlin/generated-and-checked-in/com/example/grpc/CustomerId.kt
package com.example.grpc



/** Wrapper type for typr.grpc.GrpcCodegen$$$Lambda/0x00007fc001118400@7b4c50bc */
data class CustomerId(val value: kotlin.String) {
  /** Get the underlying value */
  fun unwrap(): kotlin.String {
    return this.value
  }

  override fun toString(): kotlin.String {
    return value
  }

  companion object {
    /** Create a CustomerId from a raw value */
    fun valueOf(v: kotlin.String): CustomerId {
      return CustomerId(v)
    }
  }
}